index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<header>Guessing Number Game</header>
+<div class="position">
+<div id="text-color">Your Number:[must from 0-20]</div><input id="yournum" type="number"></input> <button id="guessbutton">Guess Now</button>
```

Comment:
When you have tags inside of tags, you want to indent the line. Also you should only have one tag per line unless its an inline tag like a span